### ISSUE TITLE
[RLlib] Ape-X DQN with custom first part net 

### ISSUE THREAD

**Issue by Jogima-cyber** [ray-project/ray#14088]
Hi there, and thanks for this beautiful RL library ! I would like to use a residual convolutional network with batch normalization (written using pytorch) as first part of the network in an Ape-X DQN model. I’ve seen in the doc and in the examples how to create and register a custom model.
However it is not clear whether my custom model will then be added a DQN head with dueling and other network related DQN advancement (if I’ve set them in the config) or if I have to implement them as part of my custom model.
